lib/stat_tracker.rb

```diff
@@ -22,7 +22,7 @@ def list_team_ids
   end
 
   def list_team_names_by_id(id)
-    @team_csv.each { |row| return row[:teamname] if id.to_s == row[:team_id] }           
+    @team_csv.each { |row| return row[:teamname] if id.to_s == row[:team_id] }
```

Comment:
This is a great helper method that we can use in many of our methods when we refactoring